.travis.yml

```diff
@@ -16,4 +16,6 @@ install:
 - bundle install
 
 script:
-- bundle exec jekyll build
+- bundle exec jekyll build 2> error.log
+- cat >&2 error.log
```

Comment:
Okay `>&2` is starting to make more sense.

But why `cat >&2 error.log` instead of `cat error.log >&2`.

Thanks for your time. I know your job is not to spend all day educating me. ;-)